tests/utils/verification/volumeproperties.go

```diff
@@ -96,6 +96,12 @@ func GetVolumePropertiesDockerCli(volName string, hostname string) string {
 	return op
 }
 
+// CheckVolumeAccessibility returns true if the given volume is accessible
+// from the specified VM; false otherwise.
+func CheckVolumeAccessibility(hostName string, volumeName string) bool {
+	return GetVolumeStatusHost(volumeName, hostName) != ""
```

Comment:
sorry it sounds confusing to me now .. `CheckVolumeAccessibility`, does this method checks whether the volume is `attached` or not?

in above comment, `docker volume ls` gives you whether volume is visible to the docker host.

can you please help me to understand here?